fpp

```diff
@@ -23,6 +23,10 @@ BASEDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 PYTHONCMD="python"
 NONINTERACTIVE=false
 
+if [ -z "$FPP_DIR" ]; then
+  FPP_DIR=~/.fpp
```

Comment:
why add this at all? we already have the default down on read (which is the better one imo). then we dont need to have two different sources of truth or places to update it